src/snowflake/cli/api/artifacts/common.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
```

Comment:
super nit - should we name this file exceptions or errors? Since we also have a utils.py file where common logic seems to be.

Reply:
this seems to be the established pattern. Look at `commands` or `entities`.